README.md

```diff
@@ -26,22 +35,39 @@ package main
 import (
 	"github.com/aws/aws-lambda-go/events"
 	"github.com/aws/aws-lambda-go/lambda"
-	wflambda "github.com/wavefronthq/wavefront-lambda-go" // Import this library
+	wflambda "github.com/retgits/wavefront-lambda-go" // Import this library
 )
 
+var wfAgent = wflambda.NewWavefrontAgent(&wflambda.WavefrontConfig{})
+
 func handler() (string, error){
 	return "Hello World", nil
 }
 
 func main() {
-	// Wrap the handler with wflambda.Wrapper()
-	lambda.Start(wflambda.Wrapper(handler))
+	// Wrap the handler with wfAgent.WrapHandler()
+	lambda.Start(wfAgent.WrapHandler(handler))
```

Comment:
Probably name this as wfAgent.Wrapper() for purpose of consistency across lambda wrappers in other languages (like go , nodejs etc.)

Reply:
Consistency is very important, so I've renamed it to Wrapper